src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Commands/Create/CreateDialogCommand.cs

```diff
@@ -74,6 +75,19 @@ public async Task<CreateDialogResult> Handle(CreateDialogCommand request, Cancel
             dialog.Org = serviceResourceInformation.OwnOrgShortName;
         }
 
+        if (request.IdempotentKey is not null && !string.IsNullOrEmpty(dialog.Org))
+        {
+            var dialogId = await _db.Dialogs
+                .Where(x => x.IdempotentKey == request.IdempotentKey && x.Org == dialog.Org).Select(x => x.Id)
+                .FirstOrDefaultAsync(cancellationToken);
+
+            if (dialogId != default)
+            {
+                return new Conflict($"IdempotencyKey: '{request.IdempotentKey}' already exists with DialogId '{dialogId}'");
```

Comment:
```suggestion
                return new Conflict(nameof(request.IdempotentKey), $"IdempotentKey '{request.IdempotentKey}' already exists with DialogId '{dialogId}'.");
```

Her må du kunne ta inn en key fra i Conflict.ctor